app/jenkins-for-jira-ui/src/components/ConnectionPanel/NotConnectedState.tsx

```diff
@@ -1,52 +1,90 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { cx } from '@emotion/css';
 import Button from '@atlaskit/button';
 import { token } from '@atlaskit/tokens';
+import Spinner from '@atlaskit/spinner';
```

Comment:
I can't use the JenkinsSpinner component we created that utilises this package because the positioning is based on the view height of the page so it ends up rendering in a completely random spot.